bacs/src/main/java/com/adyen/checkout/bacs/BacsDirectDebitComponent.kt

```diff
@@ -98,6 +99,10 @@ class BacsDirectDebitComponent internal constructor(
 
     override fun isConfirmationRequired(): Boolean = bacsDelegate.isConfirmationRequired()
 
+    override fun submit() {
+        (delegate as? ButtonDelegate)?.onSubmit() ?: Logger.e(TAG, "Active delegate can't be submitted, ignoring.")
```

Comment:
Maybe we could make this log message more understandable by merchants? As far as they are concerned there's nothing called delegate